tracer/test/benchmarks/Benchmarks.Trace/Program.cs

```diff
@@ -8,13 +8,24 @@
 using BenchmarkDotNet.Filters;
 using Benchmarks.Trace.DatadogProfiler;
 using Benchmarks.Trace.Jetbrains;
+using Benchmarks.Trace.Iast;
```

Comment:
namespace doesnt exist in the project

Reply:
Yes, you are right. This is cherrypicked from the IAST micro benchmark PR. Removing it